web-registry/src/lib/db/types/json-ld/project-metadata.ts

```diff
@@ -1,37 +1,44 @@
 import { TypeValue, UrlList, UrlType } from 'lib/rdf/types';
```

Comment:
The new interface `ProjectPageMetadataLD` on this page pulls fields from `ProjectMetadataLD`.

These changes guide where display items come from in `ProjectDetails` and its nested components.